README.md

```diff
@@ -15,30 +15,50 @@ If you have selected install_dependencies during the prompt, you can skip this p
     conda env create -f environment.yml 
     ```
 
-### Write your component code in `src/lib/components/UkGovDashComponents.react.js`.
-
-- The demo app is in `src/demo` and you will import your example component code into your demo app.
-- Test your code in a Python environment:
-    1. Build your code
-        ```
-        npm run build
-        ```
-    2. Run and modify the `example.py` sample dash app:
-        ```
-        python example.py
-        ```
-    3. Visit http://localhost:8050 in your web browser
-
-- Write tests for your component.
-    - A sample test is available in `tests/test_example.py`, it will load `example.py` and you can then automate interactions with selenium.
-    - Run the tests with `pytest tests`.
-    - The Dash team uses these types of integration tests extensively. Browse the Dash component code on GitHub for more examples of testing (e.g. https://github.com/plotly/dash-core-components)
+## Create a new component
+
+Use the `ComponentTemplate` component as a template.
+
+1. Copy `src/lib/components/ComponentTemplate.react.js` -> `src/lib/components/YourComponent.react.js`
+1. Copy `src/lib/fragments/ComponentTemplate.react.js` -> `src/lib/fragments/YourComponent.react.js`
+1. Modify the newly created files, replacing all references to `ComponentTemplate` with `YourComponent`, and replacing the sample functionality with what you'd like.
+1. Add both an import and export of `YourComponent` to both `src/lib/index.js` and `src/lib/LazyLoader.js`
+1. Add `YourComponent` to the import at the top of the `src/demo/App.js` file, and add `<YourComponent>` to the App.render() function.
+    1. Run `npm start` to start the demo server.
+    1. Open http://localhost:55555 in your browser.
+    1. If nothing apears on the page, then open up the browser console to see the error(s).
+
+## Test your code within a Dash Application
+

```

Comment:
Clarify that previously you've been writing React code which isn't connected to Python.